app/src/main/java/com/example/bookwormadventuresdeluxe2/Activities/AddOrEditBooksActivity.java

```diff
@@ -139,17 +141,6 @@ public void onClick(View view)
 
         if (this.editingBook)
         {
```

Comment:
Maybe we can disable the edit text fields here? As a user, it would suck if I typed all the changes then I was informed that I can't save my changes. PS: Something like [this](https://stackoverflow.com/questions/4297763/disabling-of-edittext-in-android)

Reply:
cool suggestion